frontend/src/features/projects/dispose/ProjectDisposeLayout.test.tsx

```diff
@@ -38,102 +39,6 @@ const loc = {
   hash: '',
 } as Location;
 
-const mockWorkflow = [
```

Comment:
this is copy pasted into a few test, trying to use the one in testutils instead.